doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>I think we should be explicit why we went for those two alternative. We should stress out that for large networks and large dataset it might be impossible to store in memory all the gradients for each example. That's why when the on the fight computation should be used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I moved the comment in the 'Notes on usage' section</p>